hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/codec/ReconOMDBDefinition.java

```diff
@@ -0,0 +1,226 @@
+      = new DBColumnFamilyDefinition<>(FILE_TABLE, StringCodec.get(), CUSTOM_CODEC_FOR_KEY_TABLE);
+
+  /** openFileTable: /volumeId/bucketId/parentId/fileName/id :- KeyInfo. */
+  public static final DBColumnFamilyDefinition<String, OmKeyInfo> OPEN_FILE_TABLE_DEF
+      = new DBColumnFamilyDefinition<>(OPEN_FILE_TABLE, StringCodec.get(), CUSTOM_CODEC_FOR_KEY_TABLE);
+
+  public static final Codec<OmDirectoryInfo> CUSTOM_CODEC_FOR_DIR_TABLE = new DelegatedCodec<>(
+      Proto2Codec.get(OzoneManagerProtocolProtos.DirectoryInfo.getDefaultInstance()),
+      ReconOMDBDefinition::getOmDirInfoFromProtobuf,
+      null,
+      OmDirectoryInfo.class);
+  /** directoryTable: /volumeId/bucketId/parentId/dirName :- DirInfo. */
+  public static final DBColumnFamilyDefinition<String, OmDirectoryInfo> DIRECTORY_TABLE_DEF
+      = new DBColumnFamilyDefinition<>(DIRECTORY_TABLE, StringCodec.get(), CUSTOM_CODEC_FOR_DIR_TABLE);
+
+  /** deletedDirectoryTable: /volumeId/bucketId/parentId/dirName/objectId :- KeyInfo. */
+  public static final DBColumnFamilyDefinition<String, OmKeyInfo> DELETED_DIR_TABLE_DEF
+      = new DBColumnFamilyDefinition<>(DELETED_DIR_TABLE, StringCodec.get(), CUSTOM_CODEC_FOR_KEY_TABLE);
+
+  // Merged column family map: OM base + Recon overrides
+  public static final Map<String, DBColumnFamilyDefinition<?, ?>> COLUMN_FAMILIES;
+
+  //---------------------------------------------------------------------------
+  static {
+    Map<String, DBColumnFamilyDefinition<?, ?>> merged =
+        new HashMap<>(OMDBDefinition.get().getMap());
+    merged.putAll(buildOverrides());
+    COLUMN_FAMILIES = Collections.unmodifiableMap(merged);
+  }
+
+  private static final ReconOMDBDefinition INSTANCE = new ReconOMDBDefinition();
+
+  /**
+   * Parses BucketInfo protobuf and creates OmBucketInfo without deserializing ACL list.
+   * @param bucketInfo
+   * @return instance of OmBucketInfo
+   */
+  public static OmBucketInfo getOmBucketInfoFromProtobuf(OzoneManagerProtocolProtos.BucketInfo bucketInfo) {
+    return OmBucketInfo.newBuilderFromProtobufPartial(bucketInfo).build();
+  }
+
+  //---------------------------------------------------------------------------
+  public static OmKeyInfo getOmKeyInfoFromProtobuf(OzoneManagerProtocolProtos.KeyInfo keyInfo) {
+    return OmKeyInfo.newBuilderFromProtobufPartial(keyInfo).build();
+  }
+
+  public static RepeatedOmKeyInfo getRepeatedOmKeyInfoFromProto(
+      OzoneManagerProtocolProtos.RepeatedKeyInfo repeatedKeyInfo) {
+    List<OmKeyInfo> list = new ArrayList<>();
+    for (OzoneManagerProtocolProtos.KeyInfo k : repeatedKeyInfo.getKeyInfoList()) {
+      list.add(ReconOMDBDefinition.getOmKeyInfoFromProtobuf(k));
+    }
+    return new RepeatedOmKeyInfo.Builder().setOmKeyInfos(list).build();
+  }
+
+  /**
+   * Parses DirectoryInfo protobuf and creates OmPrefixInfo.
+   * @param dirInfo
+   * @return instance of OmDirectoryInfo
+   */
```

Comment:
This javadoce does not provide any useful information and generate "tag description is missing" warning.  Please remove it or rewrite it.